### PR TITLE
Security and usability improvements

### PR DESCRIPTION
**Pull request by seanthegeek**
Make the template more secure and easier to use in production

- Move all config options from `publish_feed.py` to `.env`
- Change `FLASK_RUN_HOST` in `.flaskenv` from `0.0.0.0` to `127.0.0.1`
- Add `load_dotenv()` to `server/config.py` so the `.env` file is loaded in production
- Replace `WHATS_ALF` with `FEED` and `whats_alf` with `feed` to make the template more generic for general use
- Replace `logger.info()` calls with `logger.debug()` to avoid filing up the console/logs when not debugging
- Add `setupvenv.sh`
- Log level will switch from INFO to DEBUG when the flask CLI is used
- Update `README.md`

### CODE REVIEW

File: publish_feed.py [hunk 1]
```diff
@@ -2,39 +2,44 @@
...
+
+from dotenv import load_dotenv
 from atproto import Client, models
 
+load_dotenv()
+
 # YOUR bluesky handle
 # Ex: user.bsky.social
-HANDLE: str = ''
+HANDLE: str = os.environ.get("HANDLE")
```
Comment (MarshalX) [inline publish_feed.py:14]: Could we use a single quote mark here for the rest of the project?

File: server/data_filter.py [hunk 2]
```diff
@@ -21,7 +21,7 @@ def operations_callback(ops: defaultdict) -> None:
         # print all texts just as demo that data stream works
         post_with_images = isinstance(record.embed, models.AppBskyEmbedImages.Main)
         inlined_text = record.text.replace('\n', ' ')
-        logger.info(
+        logger.debug(
```
Comment (MarshalX) [inline server/data_filter.py:24]: the idea behind "info" here was about to show the user on the very first run that it works. i agree that we need an ability to easily turn it off. so let's change it to debug level as you did, but bump default logging level to DEBUG? and optionally make level configurable form the env vars?

**Review by MarshalX — COMMENTED** [review]
thank you! we should update "publishing feed" section of main README as well

File: publish_feed.py (hunk 1 above)
Comment (seanthegeek) [reply publish_feed.py]: Done

File: server/data_filter.py (hunk 2 above)
Comment (seanthegeek) [reply server/data_filter.py:24]: I'm thing there's got to be some way to check if the flask development server is running in debug mode. That way, we could keep the default log level ant infom but automatically switch the log level to debug if `flask --debug` is used.

Comment (seanthegeek) [reply server/data_filter.py:24]: Done. Now the log level will switch from INFO to DEBUG when the flask CLI is used. That should also encourage people to switch to a WSGI server like waitress.

**Pull request merged**